ajax/priority.php

```diff
@@ -43,13 +43,21 @@
     isset($_REQUEST["urgency"])
     && isset($_REQUEST["impact"])
 ) {
-    $priority = Ticket::computePriority($_REQUEST["urgency"], $_REQUEST["impact"]);
+    // Read predefined templates fields
+    $predefined_fields  = $_REQUEST["_predefined_fields"] ?? [];
+    $predefined_fields  = Toolbox::decodeArrayFromInput($predefined_fields);
+
+    // Fallback to Form value -> Template values -> Medium
+    $priority = Ticket::computePriority(
+        $_REQUEST["urgency"] ?: $predefined_fields['urgency'] ?? 3 /* Medium */,
```

Comment:
I do not understand the hardcoded `3`.

Reply:
There as no constants for priority/impact/urgency values (or I didn't found them).

Reply:
3 is the default medium value, it make more sense to use it in case of a missing value than 0.